packages/manager/modules/sms/src/sms/batches/statistics/telecom-sms-batches-statistics.html

```diff
@@ -0,0 +1,139 @@
+<header>
+    <oui-back-button on-click="$ctrl.goBack()">
+        <span data-translate="sms_batches_statistics_title"></span>
+    </oui-back-button>
+</header>
+
+<section class="container-fluid ml-3">
+    <oui-message
+        type="error"
+        data-ng-if="$ctrl.error"
+        dismissable
+        on-dismiss="$ctrl.error = null"
+    >
+        <span
+            data-translate="sms_batches_statistics_load_error"
+            data-translate-values="{ 'error': $ctrl.error }"
+        ></span>
+    </oui-message>
+
+    <div class="row">
+        <oui-tile class="col-md-8">
+            <div class="row mb-5">
+                <div class="col-md-3">
+                    <dl class="oui-description">
+                        <dt data-translate="sms_batches_name"></dt>
+                        <dd>
+                            <oui-select
+                                match="name"
```

Comment:
some missing data-

Reply:
This is strange to see some attributes with `data-` and some without